docs/pages/eas/workflows/examples.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: Example workflows
+description: Common workflows that are useful for your project.
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+The following workflows are examples of how you can use EAS Workflows to automate your development and release processes. They can help you and your team develop, review each other's PRs, and release changes to your users.
+
+## Development builds workflow
+
+[Development builds](/develop/development-builds/introduction/) are specialized builds of your project that include Expo's developer tools. These types of builds include all native dependencies inside your project, enabling you to run a production-like build of your project on a simulator, emulator, or a physical device.
+
+<Collapsible summary="Prerequisites">
+
+To get started, you'll need to configure your project and devices to build and run development builds.
+
+<BoxLink
+  title="Development build prerequisites"
+  description="Get your project ready for development builds."
+  href="/develop/development-builds/create-a-build/#prerequisites"
+  Icon={BookOpen02Icon}
+/>
```

Comment:
This could lead to some confusion and a lengthy path, I think. Because the prerequisites leed to EAS Build page. Instead can we link Android and iOS  environment sections like this one: https://docs.expo.dev/get-started/set-up-your-environment/?mode=development-build#set-up-an-android-device-with-a-development-build?

Reply:
Reading the content below, it is mostly installing EAS CLI and dev client library since the build profile stuff is already covered, maybe we can just include those steps here.

Reply:
Thanks for noticing this. I am going to include four links (ios device, android device, ios simulator, android emulator), since all of them require different steps (like having a developer account, having xcode, etc)